operations/helm/charts/mimir-distributed/templates/ruler/ruler-sa.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.ruler.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "mimir.serviceAccountName" . }}
```

Comment:
should this use `mimir.ruler.serviceAccountName`?

Reply:
Absolutely, otherwise, it wouldn't make much sense :facepalm: